### DISCUSSION TITLE
[Suggestions] Hero screen and Castle screen, allow to access the "Kingdom Overview" screens

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#7633] [Ideas]
In fheroes2, it would be nice to access the "Kingdom Overview" screens from the Hero screen and the Castle screen.

In the hero screen, when the mouse cursor is positioned over the player's emblem, "View Heroes Summary" is displayed in the information bar at the bottom of the screen.
In the hero screen, a left click on the player's emblem opens the "Heroes" screen of the "Kingdom Overview" screen.
When the player exits the "Kingdom Overview" screen, he returns to the hero screen.

In fheroes2:

![Kingdom overview fh2 01](https://user-images.githubusercontent.com/43583994/96621452-ea823500-1308-11eb-9f88-2dc356261b79.png)

![Kingdom overview fh2 02](https://user-images.githubusercontent.com/43583994/96621458-ec4bf880-1308-11eb-8a9a-c1e264a8ec54.png)

In the castle screen, when the mouse cursor is positioned over the player's emblem, "View Towns/Castles Summary" is displayed in the information bar at the bottom of the screen.
In the castle screen, a left click on the player's emblem opens the "Towns/Castles" screen of the "Kingdom Overview" screen.
When the player exits the "Kingdom Overview" screen, he returns to the castle screen.

In fheroes2:

![Kingdom overview fh2 03](https://user-images.githubusercontent.com/43583994/96621536-0685d680-1309-11eb-94bb-54cbe3aba075.png)

![Kingdom overview fh2 04](https://user-images.githubusercontent.com/43583994/96621531-0554a980-1309-11eb-829f-28f653455849.png)

In the castle screen, if a captain or a hero is present at the location of the player's emblem, a graphic is added to the top left of the frame (or other solution) to allow access to the "Heroes" screen of the "Kingdom Overview" screen.

In fheroes2:
Screens are for explanation only.

![Kingdom overview fh2 05a](https://user-images.githubusercontent.com/43583994/96621586-14d3f280-1309-11eb-88c1-0604a4d44623.png)

![Kingdom overview fh2 06a](https://user-images.githubusercontent.com/43583994/96621594-17364c80-1309-11eb-98a4-12838729e0bd.png)

Related issue: https://github.com/ihhub/fheroes2/issues/1847.

**Comment by ihhub** [comment]
I am moving this to the discussion section. This idea is not intuitive for players or requires some UI redesign which should fit flawlessly in the game.
